src/ISA/ISA.Spreadsheet/CompositeColumn.fs

```diff
@@ -0,0 +1,44 @@
+﻿module ISA.Spreadsheet.CompositeColumn
+
+open ISA
+open FsSpreadsheet
+
+
+let fromFsColumns (columns : list<FsColumn>) : CompositeColumn =
+    let header = 
+        columns
+        |> List.map (fun c -> c.[0])
+        |> CompositeHeader.fromFsCells
+    let l = columns.[0].RangeAddress.LastAddress.RowNumber
+    let cells = 
+        [|
+        for i = 1 to l do
+            columns
+            |> List.map (fun c -> c.[i])
+            |> CompositeCell.fromFsCells
+        |]                 
+    CompositeColumn.create(header,cells)
+
+
+let toFsColumns (column : CompositeColumn) : FsCell list list =
+    let hasUnit = column.Cells |> Seq.exists (fun c -> c.isUnitized)
+    let isTerm = column.Header.IsTermColumn
+    let header = CompositeHeader.toFsCells hasUnit column.Header
+    let cells = column.Cells |> Array.map (CompositeCell.toFsCells isTerm hasUnit)
```

Comment:
Okay, i don't get it. The CompositeCells must know what they are? Why try to tell them from outside?